src/guides/v2.4/install-gde/prereq/mysql.md

```diff
@@ -2,7 +2,7 @@
 group: installation-guide
 title: MySQL
 redirect_from:
-  - guides/v2.3/install-gde/trouble/tshoot_mysql_table-open-cache.html
+  - install-gde/trouble/tshoot_mysql_table-open-cache.html
```

Comment:
You can remove this redirect from the v2.4 file. It doesn't work.

Works fine for v2.3
https://devdocs.magento.com/guides/v2.3/install-gde/trouble/tshoot_mysql_table-open-cache.html -- redirects to https://devdocs.magento.com/guides/v2.3/install-gde/prereq/mysql.html

https://devdocs.magento.com/guides/v2.4/install-gde/prereq/mysql.html -- redirects to Page not found

Reply:
@meker12, I think you mean that `http://devdocs.magento.com/guides/v2.4/install-gde/trouble/tshoot_mysql_table-open-cache.html` returns a 404.

Reply:
@jeff-matthews @meker12 
Yes - as this file is not a Symlinked file anymore, it needed to have 2.4 added. 
I've changed to guides/v2.4/install-gde/trouble/tshoot_mysql_table-open-cache.html to match the format in 2.3 version of file